Gopkg.toml

```diff
@@ -73,6 +73,10 @@
   name = "github.com/tendermint/go-amino"
   version = "~0.10.1"
 
+[[constraint]]
+  name = "github.com/tendermint/iavl"
```

Comment:
Can we exclude IAVL dependency from tendermint? IAVL repo will be moved into the SDK, making dependency cycle. The only part in this code that uses IAVL directly is abci example, which can be reverted or moved into the SDK. @Liamsi @ebuchman

Reply:
Talked to bucky recently and figured we'll move it into tendermint/tendermint because it's useful to have them for the abci/example/kvstore app.